app/javascript/readingList/readingList.jsx

```diff
@@ -189,14 +186,14 @@ export class ReadingList extends Component {
       ''
     );
     return (
-      <div>
+      <section>
         <header className="crayons-layout flex justify-between items-center pb-0">
           <h1 class="crayons-title">
             {isStatusViewValid ? 'Reading list' : 'Archive'}
-            {` (${totalCount > 0 ? totalCount : '0'})`}
+            {` (${items.length})`}
           </h1>
 
-          <div class="flex items-center">
+          <fieldset class="flex items-center">
```

Comment:
(blocking) - I don't think this should be a `fieldset` - fieldsets are typically used for grouping related inputs e.g. a specific section of a form or multiple radio options. The 'view archive' and 'search' functions I don't think fit this criteria, and they're only really grouped together for visual purposes so I think `div` is ok